moto/ec2/responses/spot_fleets.py

```diff
@@ -89,6 +91,16 @@ def request_spot_fleet(self) -> str:
         <item>
             <spotFleetRequestId>{{ request.id }}</spotFleetRequestId>
             <spotFleetRequestState>{{ request.state }}</spotFleetRequestState>
+            {% for resource_type in request.tags %}
+            <tag>
+                {% for key, value in request.tags[resource_type].items() %}
```

Comment:
This should be modified to only pull in tags for the resource type you want.  Something like this:

`{% for key, value in request.tags.get('spot-fleet-request',{}).items() %}`